Frontend/src/app/components/newest-prices-list/newest-prices-list.component.spec.ts

```diff
@@ -1,18 +1,19 @@
 import {ComponentFixture, TestBed} from '@angular/core/testing';
 
 import {NewestPricesListComponent} from './newest-prices-list.component';
-import {RouterTestingModule} from "@angular/router/testing";
-import {HttpClient} from "@angular/common/http";
-import {AbstractMockObservableService} from "../../mocks/mock.service";
-import {ApiService} from "../../services/api.service";
+import {RouterTestingModule} from '@angular/router/testing';
+import {HttpClient} from '@angular/common/http';
+import {AbstractMockObservableService} from '../../mocks/mock.service';
+import {ApiService} from '../../services/api.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Mueller-Patrick/Betterzon/pullRequest?prid=7401082)
```suggestion
import {ApiService} from "../../services/api.service";
```